lib/matplotlib/image.py

```diff
@@ -232,6 +234,23 @@ def get_size(self):
 
         return self._A.shape[:2]
 
+    def get_antialiased(self):
+        """
+        Get whether down-sampled images are low-pass filtered before
+        down-sampling (i.e. when there is more data than pixels).
```

Comment:
docstring needs update

Reply:
Yes, I've not done anything to properly document this yet